src/main/groovy/com/okta/test/mock/scenarios/ImplicitRemoteValidationScenarioDefinition.groovy

```diff
@@ -54,7 +54,7 @@ class ImplicitRemoteValidationScenarioDefinition implements ScenarioDefinition {
                     .withRequestBody(matching(".*token=some.random.jwt.*"))
                         .willReturn(aResponse()
                         .withHeader("Content-Type", "application/json;charset=UTF-8")
-                        .withBodyFile("introspect.json")))
+                        .withBodyFile("introspect-groups.json")))
```

Comment:
was any other teat using this endpoint?

Reply:
No other test is impacted with this change. I have verified it.

Reply:
@bdemers any followup comments on this?